Libraries/Drei/lisp-syntax.lisp

```diff
@@ -50,6 +50,8 @@
 ;;;
 ;;; the syntax object
 
+(declaim (special |initial-state |))
```

Comment:
I honestly am not sure how declaim with special works, but I'd assume that results in a global special declaration, similar to how `DEFVAR` works?

Reply:
Yes, that's what I think as well.

Reply:
please make it defvar too, while it may be that they are the same it will be less surprising to read defvar and there would no such discussion in the first place (what kind of proves the point)